euclidean_gcd/C/euclidean_gcd.c

```diff
@@ -0,0 +1,47 @@
+#include <stdio.h>
+/*
+Calculates GCD of two numbers a & b using the division-based Euclidean Algorithm
+*/
+int gcd_div(int a , int b) {
+    while(b != 0) {
+        int temp = b;
```

Comment:
Missing space before ( in while(  [whitespace/parens] [5]

*Origin: CPPLintBear, Section: `all.cpp`.*